src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -14,26 +15,41 @@ interface IGenomicOverviewProps {
     sampleColors: {[s:string]:string};
     sampleManager: SampleManager;
 }
-export default class GenomicOverview extends React.Component<IGenomicOverviewProps, {vafPlotData:{[s:string]:number[]}}> {
+export default class GenomicOverview extends React.Component<IGenomicOverviewProps, { vafPlotData:{[s:string]:number[]}}> {
+
+    shouldComponentUpdate(){
+        return false;
+    }
 
     constructor(props:IGenomicOverviewProps) {
         super(props);
         const vafPlotData = this.computeVAFPlotData(props.mutations);
         this.state = {
             vafPlotData,
         };
+
+    }
+
+    componentDidMount(){
+
+        var debouncedResize =  _.debounce(()=>this.forceUpdate(),500);
+
+        $(window).resize(debouncedResize);
+
     }
 
     public render() {
 
+        const width = $('.tab-content').width()-140;
```

Comment:
it would be better if this was the size of this components outer div, but i can't do that because it will be zero when tab is not visible